src/airdecloak-ng/airdecloak-ng.c

```diff
@@ -560,7 +560,7 @@ static BOOLEAN read_packets(void)
 		}
 
 #ifdef DEBUG
-		printf("From DS: %d - ToDS: %d\n",
+		printf("From DS: %d - To DS: %d\n",
```

Comment:
Actually, ToDS is correct, and that's "From DS" that should be changed to "FromDS"